frontend/ui/widget/opdsbrowser.lua

```diff
@@ -504,7 +504,7 @@ function OPDSBrowser:downloadFile(item, format, remote_url)
     logger.dbg("downloading file", local_path, "from", remote_url)
 
     local parsed = url.parse(remote_url)
-    http.TIMEOUT, https.TIMEOUT = 10, 10
+    http.TIMEOUT, https.TIMEOUT = 20, 20
     local httpRequest = parsed.scheme == 'http' and http.request or https.request
     local _, c, _ = httpRequest{
```

Comment:
We can do `local gettext = require("gettext")` for this module, then do `gettext("abc")`, which is also a very common practice.